src/main/java/ch/wisv/areafiftylan/extras/consumption/model/ConsumptionMap.java

```diff
@@ -37,7 +38,7 @@
 
     @NonNull
     @ManyToMany(fetch = FetchType.EAGER)
-    private Collection<Consumption> consumptionsMade;
+    private List<Consumption> consumptionsMade;
```

Comment:
@skamoen So now here a list is instantiated and somewhere else a collection? What if you change that collection to a list? Doesn't the ambiguity return?